src/drive/targets/mobile/InitAppMobile.jsx

```diff
@@ -153,7 +153,7 @@ class InitAppMobile {
     if (window.plugins && window.plugins.intentShim) {
       window.plugins.intentShim.onIntent(intentHandlerAndroid(store))
       window.plugins.intentShim.getIntent(intentHandlerAndroid(store), err => {
-        console.error('Error getting launch intent', err)
+        logger.error('Error getting launch intent', err)
```

Comment:
I think that this syntax works for minilog too?

Reply:
it works